Content.Server/Atmos/Monitor/Systems/AtmosMonitoringSystem.cs

```diff
@@ -56,8 +60,15 @@ private void OnAtmosDeviceLeaveAtmosphere(EntityUid uid, AtmosMonitorComponent a
 
     private void OnAtmosDeviceEnterAtmosphere(EntityUid uid, AtmosMonitorComponent atmosMonitor, ref AtmosDeviceEnabledEvent args)
     {
+        if (atmosMonitor.MonitorsPipeNet)
+        {
+            atmosMonitor.TileGas = GetGasPipeAirMixture(uid, atmosMonitor);
+            return;
```

Comment:
```suggestion
        if (atmosMonitor.MonitorsPipeNet && _nodeContainerSystem.TryGetNode<PipeNode>(uid, atmosMonitor.NodeNameMonitoredPipe, out var pipeNode))
        {
            atmosMonitor.TileGas = pipeNode.Air;
            return;
```
No need for a helper function for this. If it leaves the Atmosphere TileGas is set back to null anyways.